src/lib/theme.js

```diff
@@ -0,0 +1,43 @@
+const BASE_MEASURE = 8
+
+const spacing = (n) => n * BASE_MEASURE
+
+const shadow = () => ({
+    shadowColor: 'rgb(212, 221, 232)',
+    shadowOffset: {
+        width: 0,
+        height: 2,
+    },
+    shadowOpacity: 0.4,
+    shadowRadius: 15,
+    elevation: 1,
+})
+
+const colors = {
+    primary: '',
+    secondary: '',
+    background: '',
+    white: '',
+    black: '',
+}
+
+const fontSizes = {
+    h1: 96,
```

Comment:
Used material design font size rules and types. Happy to get feedback and change.

Reply:
I think this is fine for now.

Reply:
We can revise in the future in response to the potential objections that'll come from the design department (!)